examples/orders_create_and_download_multiple_orders.py

```diff
@@ -50,33 +59,34 @@
                      [-117.558734, 45.229745]]]
 }
 
-oregon_images = ['20200909_182525_1014', '20200909_182524_1014']
+oregon_items = ['20200909_182525_1014', '20200909_182524_1014']
```

Comment:
Whoopsie, the `images` suffix was from my old days in astronomy, thanks for catching this.